### PR TITLE
Add LICENSE and licensed config

### PR DESCRIPTION
**Pull request by richerve**

### CODE REVIEW

File: .licensed.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+sources:
```
Comment (yuzhouliu9) [inline .licensed.yaml:1]: Do we need to add this file if `.drone.yaml` doesn't have the license-check step?
Or perhaps you were looking to add it in later?

Comment (richerve) [reply .licensed.yaml:1]: my bad, let me add the step in the pipeline

Comment (richerve) [reply .licensed.yaml:1]: done

**Review by yuzhouliu9 — COMMENTED** [review]
If I'm not mistaken by commenting out the pipeline, then drone pipeline wouldn't run, right? We definitely need the drone pipeline to run.

File: .drone.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 kind: pipeline
-type: kubernetes
+# type: kubernetes
```
[the remark this answers is not in the document]
Comment (richerve) [reply .drone.yml]: fixed

**Review by yuzhouliu9 — APPROVED** [review]

**Pull request merged**